blog/_posts/2021-09-02-kubefwd-operator.md

```diff
@@ -0,0 +1,196 @@
+---
+slug: "kubefwd-operator"
+date: 2021-09-02
+author: nick
+layout: blog
+title: "How to Automagically Setup `kubefwd` to Bulk-Forward Ports"
+subtitle: "A walkthrough of Tilt's kubefwd operator"
+description: "A walkthrough of Tilt's kubefwd operator"
+image: "/assets/images/kubefwd-operator/bulk.jpg"
+image_caption: "A truck bulk-forwarding glass, plastic, and metal. Kubernetes networking isn't trash! From Tilt Dev's field trip to the <a href='https://en.wikipedia.org/wiki/Sunset_Park_Material_Recovery_Facility'>Sunset Park Material Recovery Facility.</a>"
+tags:
+  - api
+  - bash
+  - extensions
+  - kubefwd
+---
+
+[`kubefwd`](https://kubefwd.com/) is a great tool for both getting started with
```

Comment:
Rearrangement suggestion:

> `kubefwd` is a tool that makes network magic so you can access services from localhost the same way you would from inside a cluster. For example, I could access `http://elasticsearch:9200/` directly from localhost—something that would normally only work for services within the cluster.
> It's a great tool for getting started with Kubernetes. We even recommend it for folks [...]
> But it can be hard to [...]

Reply:
done